charts/tekton-pipeline/templates/pipelineruns.tekton.dev-crd.yaml

```diff
@@ -87,40 +87,10 @@ spec:
       # starts to increment
       subresources:
         status: {}
-    - name: v1
```

Comment:
This was removed in this commit: https://github.com/tektoncd/pipeline/commit/2777b19d6486e66e9e97f94b7078b0f129e0ee52